spec/parsers/rw2_parser_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+describe FormatParser::RW2Parser do
+  context 'when likely_match? is called' do
+    %w[rw2 RW2 raw RAW rwl RWL].each do |extension|
+      context "with a file with a .#{extension} extension" do
+        it "should return true" do
```

Comment:
```suggestion
        it "returns true" do
```

as per [better specs](https://www.betterspecs.org/).